stack.yaml

```diff
@@ -0,0 +1,61 @@
+resolver: lts-5.13
```

Comment:
[The wiki](https://wiki.haskell.org/Diagrams/Contributing#Keeping_up_to_date) describes how I run a fresh checkout & keep all the repos up to date.  My [complete .mrconfig](https://github.com/bergey/diagrams-stow/blob/master/mr/.mrconfig) has even more Diagrams packages.  I think the wiki is hard for new contributors to find, though.